shifthelper/qla.py

```diff
@@ -0,0 +1,186 @@
+import pandas as pd
+import numpy as np
+import matplotlib
+
+from .tools import create_db_connection
+
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+from matplotlib.dates import DateFormatter
+
+database = create_db_connection()
+
+
+def dorner_binning(data, bin_width_minutes=20):
```

Comment:
there is a function called `dorner_binning(data, bin_width_minutes=20)` in https://github.com/fact-project/plot_qla/blob/master/plot_qla.py, why is this reproduced here?